### DISCUSSION TITLE
Please help me, I can't find my precious account!

### DISCUSSION THREAD

**Discussion by xk910** [community/community#23529] [New to GitHub]
<p>My user name: xk730, I have been unable to log in to him for a long time! Because I forgot my email; I tried various methods, including the API function of GitHub, but I couldn’t know what mailbox I used when I registered. Please help me, thank you! This account (xk730) is really important to me. I need to use it to invest in project development. I would appreciate it if you could help me</p>
<p>{<br>
“login”: “xk730”,<br>
“id”: 40309144,<br>
“node_id”: “MDQ6VXNlcjQwMzA5MTQ0”,<br>
“avatar_url”: “<a href="https://avatars.githubusercontent.com/u/40309144?v=4" rel="noopener nofollow ugc">https://avatars.githubusercontent.com/u/40309144?v=4</a>”,<br>
“gravatar_id”: “”,<br>
“url”: “<a href="https://api.github.com/users/xk730" rel="noopener nofollow ugc">https://api.github.com/users/xk730</a>”,<br>
“html_url”: “<a href="https://github.com/xk730" class="inline-onebox" rel="noopener nofollow ugc">xk730 · GitHub</a>”,<br>
“followers_url”: “<a href="https://api.github.com/users/xk730/followers" rel="noopener nofollow ugc">https://api.github.com/users/xk730/followers</a>”,<br>
“following_url”: “<a href="https://api.github.com/users/xk730/following%7B/other_user%7D" rel="noopener nofollow ugc">https://api.github.com/users/xk730/following{/other_user}</a>”,<br>
“gists_url”: “<a href="https://api.github.com/users/xk730/gists%7B/gist_id%7D" rel="noopener nofollow ugc">https://api.github.com/users/xk730/gists{/gist_id}</a>”,<br>
“starred_url”: “<a href="https://api.github.com/users/xk730/starred%7B/owner%7D%7B/repo%7D" rel="noopener nofollow ugc">https://api.github.com/users/xk730/starred{/owner}{/repo}</a>”,<br>
“subscriptions_url”: “<a href="https://api.github.com/users/xk730/subscriptions" rel="noopener nofollow ugc">https://api.github.com/users/xk730/subscriptions</a>”,<br>
“organizations_url”: “<a href="https://api.github.com/users/xk730/orgs" rel="noopener nofollow ugc">https://api.github.com/users/xk730/orgs</a>”,<br>
“repos_url”: “<a href="https://api.github.com/users/xk730/repos" rel="noopener nofollow ugc">https://api.github.com/users/xk730/repos</a>”,<br>
“events_url”: “<a href="https://api.github.com/users/xk730/events%7B/privacy%7D" rel="noopener nofollow ugc">https://api.github.com/users/xk730/events{/privacy}</a>”,<br>
“received_events_url”: “<a href="https://api.github.com/users/xk730/received_events" rel="noopener nofollow ugc">https://api.github.com/users/xk730/received_events</a>”,<br>
“type”: “User”,<br>
“site_admin”: false,<br>
“name”: null,<br>
“company”: null,<br>
“blog”: “”,<br>
“location”: null,<br>
“email”: null,<br>
“hireable”: null,<br>
“bio”: null,<br>
“twitter_username”: null,<br>
“public_repos”: 0,<br>
“public_gists”: 0,<br>
“followers”: 2,<br>
“following”: 0,<br>
“created_at”: “2018-06-16T01:24:07Z”,<br>
“updated_at”: “2020-07-06T12:20:44Z”<br>
}</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there <a class="mention" href="/xk910">@xk910</a> :wave: I’m sorry to hear that. Have you tried contacting our <a href="https://support.github.com/contact">Support team</a>? They are best able to assist in cases of account-specific issues like this.</p>

**Comment by ghost** [comment]
i cannot find @vos-ter account yet i had some repositories inside
